g2p/mappings/utils.py

```diff
@@ -863,6 +976,16 @@ class _MappingModelDefinition(BaseModel):
     rules_path: Optional[Path] = None
     """A path to a file of a list of rules"""
 
+    hf_model_identifier: Optional[str] = None
+    """The identifier for a model hosted on HuggingFace, if applicable"""
+
+    hf_model_filepath: Optional[str] = None
+    """The path to the specific model on HuggingFace if downloading the entire repo is not needed"""
```

Comment:
Question: this seems to lock us into models that exist on HF and nowhere else. Is that where you expect all models to live for the foreseeable future?

I could imagine a more general solution with `model_host=huggingface` and then `model_identifier` and `model_filepath`, but I maybe this is unlikely to be what we really need in the future when we diversify sources anyway, I'm not sure.

So let's YAGNI and let this stand as is if you think it'll do for at least a while and you don't already anticipate other specific hosting services, but I still feel the need to ask.